geoh5py/objects/block_model.py

```diff
@@ -69,8 +69,9 @@ def local_axis_centers(self, axis: str) -> np.ndarray:
 
         :param axis: Axis to get the centers for.
         """
-        n_cells = getattr(self, f"{axis}_cells")
-        return np.cumsum(n_cells) - n_cells / 2.0
+        out = np.cumsum(getattr(self, f"{axis}_cell_delimiters"))
+        out[2:] = out[2:] - out[:-2]
+        return out[2 - 1 :] / 2
```

Comment:
`[2-1:]`? Is that not just `[1:]`?
Could also do it in one go with a convolution.

`return np.convolve(getattr(self, f"{axis}_cell_delimiters"), np.ones(2)/2)[1:-1]`

Reply:
whoops, I adapted an n-length window to length two and didn't notice the simplification.  I'll stick with the cumsum implementation because it should be more efficient